model/src/main/kotlin/config/ScannerConfiguration.kt

```diff
@@ -24,7 +24,12 @@ package com.here.ort.model.config
  */
 data class ScannerConfiguration(
         /**
-         * Configuration of the scan results Artifactory cache.
+         * (optional) Artifactory configuration to cache the scan results
```

Comment:
Please do not mention `(optional)` like this here, we prefer "prose" that could also be printed as a book (including a trailing dot at the end of the sentence). Something like:

    An optional cache configuration to store scan results in Artifactory.